actions/createProposal.ts

```diff
@@ -67,6 +68,8 @@ export const createProposal = async (
   proposalIndex: number,
   instructionsData: InstructionDataWithHoldUpTime[],
   isDraft: boolean,
+  isMulti: boolean,
+  options: string[],
```

Comment:
if you wanted to be fancy you could use `[string, ...string[]]` instead of `string[]` to show that there must be at least one option.

Reply:
probably more trouble than its worth because convincing TS that an array has members might be annoying. anyway just do it if it sounds fun.

Reply:
Removed `isMulti` and validated with `options` array. If this array has size greater than 1 then the proposal is considered multi-choice.